examples/tv-app/android/java/AppImpl.h

```diff
@@ -127,11 +142,11 @@ class DLL_EXPORT ContentAppFactoryImpl : public ContentAppFactory
     CHIP_ERROR ConvertToPlatformCatalogVendorApp(const CatalogVendorApp & sourceApp, CatalogVendorApp * destinationApp) override;
 
 protected:
-    ContentAppImpl mContentApps[APP_LIBRARY_SIZE] = {
-        ContentAppImpl("Vendor1", 1, "exampleid", 11, "Version1", "34567890"),
-        ContentAppImpl("Vendor2", 65521, "exampleString", 32768, "Version2", "20202021"),
-        ContentAppImpl("Vendor3", 9050, "App3", 22, "Version3", "20202021"),
-        ContentAppImpl("TestSuiteVendor", 1111, "applicationId", 22, "v2", "20202021")
+    std::vector<ContentAppImpl> mContentApps{
+        ContentAppImpl("Vendor1", 1, "exampleid", 11, "Version1", "34567890", nullptr),
```

Comment:
don't these nullptr arguments cause problems in the ContentAppEndpointManagerDelegate constructor?